Dockerfile.rockylinux-10

```diff
@@ -0,0 +1,17 @@
+FROM docker.io/rockylinux:10
```

Comment:
This should refer to image [10.0.20250606](https://hub.docker.com/layers/rockylinux/rockylinux/10.0.20250606/images/sha256-839c8620a6bd5e3afc884243eeb59c0741e9ffc0a6f75c6ccee020afd1308cce)